opencog/haskell/OpenCog/AtomSpace/Types.hs

```diff
@@ -156,6 +159,12 @@ data Atom (a :: AtomType) where
 
     BindLink            :: (v <~ VariableT,p <~ AtomT,q <~ AtomT) =>
                            Atom v -> Atom p -> Atom q -> Atom BindT
+    ContextLink         :: (c <~ AtomT,e <~ AtomT) =>
+                           TVal -> Atom c -> Atom e -> Atom ContextT
+    LambdaLink		:: (v <~ VariableT,a <~ AtomT) =>
+			   Atom v -> Atom a -> Atom LambdaT
+    SubsetLink          :: (a1 <~ AtomT, a2 <~ AtomT) =>
+			   TVal -> Atom a1 -> Atom a2 -> Atom SubsetT
```

Comment:
Same remark here, please try to keep the same formatting, otherwise it's gonna be a source of confusion and distraction.